src/melee/it/items/ittomato.c

```diff
@@ -0,0 +1,186 @@
+#include "ittomato.h"
+
+#include "baselib/jobj.h"
+#include "it/inlines.h"
+#include "it/it_266F.h"
+#include "it/it_26B1.h"
+#include "it/item.h"
+
+#include <placeholder.h>
+
+extern M2C_UNK gm_80473A18;
```

Comment:
This goes in its header file, `src/melee/gm/gm_1601.h`. You can use `UNK_T`, which expands to `void*` most of the time and when using M2C it expands to `M2C_UNK`.

Reply:
Oh, and don't worry about the address comments if you don't feel like it. If you happen to use vim, this is my replacement for that:
```
'<,'>s`.\{-}80\(\x\{6\}\).\+`/* \1 */ \0`e | nohl
```

And then I sort with
```
'<,'>!tools/sort-declarations.sh
```